pages/common/telnet.md

```diff
@@ -1,19 +1,19 @@
 # telnet
 
-> Telnet is used to connect to a specified port of a host.
+> Connect to a specified port of a host using the telnet protocol.
 
 - Telnet to a certain port:
 
 `telnet  {{ip_address}} {{port}}`
 
-- To exit a telnet session:
+- Exit a telnet session:
 
 `quit`
 
-- Default escape character:
+- Emit the default escape character:
 
 `CTRL + ]`
```

Comment:
"session escape character"? "session termination character"? In fact, I'm not sure it's intruitive to call this a character. Maybe "character combination", "keyboard shortcut", or something to that effect?